driver-core/src/main/com/mongodb/internal/connection/SocketStreamFactory.java

```diff
@@ -53,8 +54,18 @@ public SocketStreamFactory(final InetAddressResolver inetAddressResolver, final
         this.sslSettings = notNull("sslSettings", sslSettings);
     }
 
+    /**
+     * @see SocketStreamFactorySubstitution#create(ServerAddress)
+     */
     @Override
     public Stream create(final ServerAddress serverAddress) {
+        return createInternal(serverAddress);
```

Comment:
There is no way to call the original implementation of a method that is being substituted. I had to introduce `createInternal` as a work-around.

Reply:
Yeah, something you really want this feature.... I follow the same pattern very (too) often.